### PR TITLE
Replaced the selectbox with native HTML <select>

### PR DESCRIPTION
**Pull request by RCMainak**

### CODE REVIEW

File: app/templates/selectbox/index.js [hunk 1]
```diff
@@ -6,54 +6,28 @@ module.exports = function(selected, options, translate, changed) {
...
-      ul.classList.add('selectbox__options--active');
-      const body = document.querySelector('body');
-      body.addEventListener('click', close);
-    }
-  }
+    </select>
+    <svg id="arrow" width="32" height="32">
+      <polygon points="8 18 17 28 26 18" fill="#0094fb"/>
+    </svg>
+  </span>`;
```
Comment (dannycoates) [inline app/templates/selectbox/index.js]: I'd suggest using `onchange` for the event handler, and `selected` on the selected option. Something like:

```js
html`
  <span class="select">
    <select id="${id}" onchange=${choose}>
      ${options.map(
        i =>
          html`<option value="${i}" ${i === selected ? 'selected' : ''}>${translate(i)}</option>`
      )}
    </select>
  </span>`;
```

**Comment by dannycoates** [comment]
Great progress so far! Probably just more tweaking of the css to go. I'll be able to help out in another few days if you get stuck.

File: app/templates/selectbox/index.js (hunk 1 above)
Comment (RCMainak) [reply app/templates/selectbox/index.js]: Of course, that is much more clearer.

**Comment by dannycoates** [comment]
@RCMainak I see you've made changes. Are you done and ready for review?

**Comment by RCMainak** [comment]
Yes, I made some changes. It's working properly but not sure about CSS. Please make recommendations, I'll make changes if needed.

**Comment by dannycoates** [comment]
Thanks @RCMainak! I landed your changes after a couple tweaks on master.

**Pull request closed without merging**